apps/ckit-app/src/hooks/useSendIssueTx.tsx

```diff
@@ -0,0 +1,68 @@
+import { Address, HexNumber, Hash } from '@ckb-lumos/base';
+import { Modal } from 'antd';
+import { MintSudtBuilder, RecipientOptions } from 'ckit';
+import React from 'react';
+import { useMutation, UseMutationResult } from 'react-query';
+import { useConfigStorage } from './useConfigStorage';
+import { CkitProviderContainer, WalletContainer } from 'containers';
+import { hasProp } from 'utils';
+
+export interface SendIssueTxInput {
+  recipient: Address;
+  amount: HexNumber;
+  operationKind: 'invite' | 'issue';
+  setModalVisible: (visible: boolean) => void;
+}
+
+export function useSendIssueTx(): UseMutationResult<{ txHash: Hash }, unknown, SendIssueTxInput> {
+  const { selectedWallet } = WalletContainer.useContainer();
+  const ckitProvider = CkitProviderContainer.useContainer();
+  const [localConfig] = useConfigStorage();
+
+  return useMutation(
+    ['sendIssueTx'],
+    async (input: SendIssueTxInput) => {
+      if (!selectedWallet?.signer) throw new Error('exception: signer not set');
```

Comment:
```suggestion
      if (!selectedWallet?.signer) throw new Error('Signer could not be found, the wallet may be disconnected, you can try to reconnect');
```
is the error message for user? if so we can change it to the above. I think a good error message to end user should be cause + solution

Reply:
fixed